src/hotel/components/HotelsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import {
+    Button,
+    IconButton,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+} from "@mui/material";
+
+import EditIcon from '@mui/icons-material/Edit';
+
+import { Hotel } from "../types";
+import { EditModal } from "./EditModal";
+
+export const HotelsTable = () => {
+
+    // TODO: Implement Redux
+    const hotelData: Hotel[] = []
+
+    const [open, setOpen] = useState(false);
+
+    const handleOpen = (row: Hotel) => setOpen(true);
```

Comment:
Why did you point here row as argument if you it's not needed here

Reply:
I am setting data for modal here. Just missed it in this PR